selenium/pageObject_model/CalculatorCompletedForm.js

```diff
@@ -0,0 +1,115 @@
+import pkg from 'selenium-webdriver';
+const { Builder, Browser, By, until, wait } = pkg;
+import { CalculatorSitePage } from './CalculatorSitePage.js';
+
+export class CalculatorCompletedForm extends CalculatorSitePage {
+    constructor(driver) {
+        super(driver);
+        this.completedFormXpath = '//*[@id="resultBlock"]'; //*
+        this.VMClassXpath = '//*[@id="compute"]/md-list/md-list-item[4]/div';
+        this.instanceTypeXpath =
+            '//*[@id="compute"]/md-list/md-list-item[5]/div[1]';
+        this.regionXpath = '//*[@id="compute"]/md-list/md-list-item[1]/div';
+        this.localSsdXpath =
+            '//*[@id="compute"]/md-list/md-list-item[7]/div[1]';
+        this.termXpath = '//*[@id="compute"]/md-list/md-list-item[3]/div';
+        this.priceXpath =
+            '//*[@id="resultBlock"]/md-card/md-card-content/div/div/div/h2/b';
```

Comment:
Please investigate how to replace this locators.

Reply:
done